Mogakco/Sources/Presentation/Chat/ViewModel/ChatListViewModel.swift

```diff
@@ -25,6 +25,7 @@ final class ChatListViewModel: ViewModel {
     
     struct Output {
         let chatRooms: Driver<[ChatRoom]>
+        let loadingFinished: Observable<Bool>
```

Comment:
```suggestion
        let chatRooms: Driver<[ChatRoom]>
        let loadingFinished: Signal<Bool>
```
요기도 Trait을 적용하면 좋을 것 같습니다!